test/unit_tests/braket/circuits/test_circuit.py

```diff
@@ -799,6 +799,24 @@ def test_from_ir_with_measure():
     assert Circuit.from_ir(source=ir.source, inputs=ir.inputs) == expected_circ
 
 
+def test_from_ir_with_single_measure():
```

Comment:
Could we add tests to do a round trip transformation? `Circuit -> OQ3 -> Circuit.from_ir` and assert that the original and the parsed circuit are the same

Reply:
I will add a round trip test.

This `test_from_ir_with_single_measure` test would not have the same OpenQASM output though because `Circuit.from_ir` would not output `b = measure q` since the BDK does not handle the measure assignment. 

So  instead it would output:
```
b[0] = measure q[0];
b[1] = measure q[1];
```